ci/pipelines/pr.yml

```diff
@@ -0,0 +1,101 @@
+resource_types:
+- name: pull-request
+  type: registry-image
+  source:
+    repository: teliaoss/github-pr-resource
+
+resources:
+  - name: pull-request
+    type: pull-request
+    check_every: 1m
```

Comment:
We won't run out of github requests if we do this every minute will we?

Reply:
1m is usually the default anyway, and if it's a public repo it should be OK.